### PR TITLE
Text Summarizer with HuggingFace/Facebook BART model

### PR DESCRIPTION
**Pull request by shanepeckham**
## Purpose

* This PR adds a PowerSkill to Summarize the text in a document and includes Azure Text Analytics NER

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Everything can be tested within the jupyter notebook section

## Other Information
Every step is documented in the README.md and jupyter notebook

### CODE REVIEW

File: Text/TextSummarization/README.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+## How to implement
+
+This section describes how to get this working on sample data and how it can be amended for your data.
+ 
+1) ###Data
+   The first step is to view the sample data files here [train data](data/). 
+2) ###Run the API
+   The next step is to run the API locally and test the model against a test record. Create a local python environment
+   and install the requirements:
+   ```pythonn
```
Comment (bleroy) [inline Text/TextSummarization/README.md:43]:
```suggestion
   ```python
```

File: Text/TextSummarization/README.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+      python -m pip install -r requirements.txt
```
Comment (bleroy) [inline Text/TextSummarization/README.md:44]:
```suggestion
  python -m pip install -r requirements.txt
```

File: Text/TextSummarization/README.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+   ```
+   Activate your environment and run the API locally, execute the following:
+   ```python 
+
```
Comment (bleroy) [inline Text/TextSummarization/README.md:48]:
```suggestion
```

File: Text/TextSummarization/README.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+    ```bash
+    docker run -it --rm -p 5000:5000 -e DEBUG=true -e KEY=[YourSecretKeyCanBeAnything] 
+    -e SUMMARIZER_MODEL=facebook/bart-large-cnn -e MAX_LENGTH=1024 -e NUM_BEAMS=4 
+    [container_registry_name.azurecr.io/text_summarization_extractor:[your_tag]
+    ```
+    Upon starting you will see the download of the BART model initiate. See below for example:
+    
+    ```bash
+    DEBUG:urllib3.connectionpool:https://cdn-lfs.huggingface.co:443 "GET /facebook/bart-large-cnn/2ac2745c02ac987d82c78a14b426de58d5e4178ae8039ba1c6881eccff3e82f1 HTTP/1.1" 200 1625270765
+    Downloading:   1%|███▉
```
Comment (bleroy) [inline Text/TextSummarization/README.md:90]:
```suggestion
    Downloading:   1%
```

**Review by bleroy — CHANGES_REQUESTED** [review]
Thanks for another great contrib!

File: Text/TextSummarization/README.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+* An Azure Blob storage container
+* A provisioned Azure Cognitive Search (ACS) instance 
+* A provisioned Azure Container Registry
+* A Cognitive Services key in the region you deploy ACS to
+
+Below is a full working example that you can get working end to end on sample data.
+
+## High level Process
+
+![text summarization process](images/text_summarization_flow.png)
```
[the remark this answers is not in the document]
Comment (shanepeckham) [reply Text/TextSummarization/README.md:32]: Added

File: Text/TextSummarization/README.md (hunk 3 above)
Comment (shanepeckham) [reply Text/TextSummarization/README.md:48]: Removed

File: Text/TextSummarization/README.md (hunk 4 above)
Comment (shanepeckham) [reply Text/TextSummarization/README.md:90]: Changed

**Pull request merged**